src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1567,11 +1650,12 @@ class NodeUtils(object):
         # Now try to construct a dictionary with hyperthread information
         # if this is an Intel based processor
         try:
-            if 'Intel' in cpuinfo['cpu'][0]['vendor_id']:
+            if (('Intel' in cpuinfo['cpu'][0]['vendor_id'])
+                    or ('AuthenticAMD' in cpuinfo['cpu'][0]['vendor_id'])):
                 if 'ht' in cpuinfo['cpu'][0]['flags']:
                     cpuinfo['hyperthreads_per_core'] = \
-                        cpuinfo['cpu'][0]['siblings'] / \
-                        cpuinfo['cpu'][0]['cpu cores']
+                        (cpuinfo['cpu'][0]['siblings']
+                         / cpuinfo['cpu'][0]['cpu cores'])
```

Comment:
PEP8 disagrees with your suggestion of putting the binary operator at the end of the line to avoid parentheses:

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

To solve this readability problem, mathematicians and their publishers follow the opposite convention. Donald Knuth explains the traditional rule in his Computers and Typesetting series: "Although formulas within a paragraph always break after binary operations and relations, displayed formulas always break before binary operations" [3].

Following the tradition from mathematics usually results in more readable code:

# Yes: easy to match operators with operands
income = (gross_wages
          + taxable_interest
          + (dividends - qualified_dividends)
          - ira_deduction
          - student_loan_interest)